source/Libraries/Iciclecreek.Bot.Builder.Dialogs.Recognizers.Lucene/Schemas/en-us/LuceneDynamicListEntity.en-us.lg.lg

```diff
@@ -0,0 +1,24 @@
+[standard.en-us.lg](standard.en-us.lg)
+
+# filename
+- ${entityLGFilename()}
+
+# template
+-```
+${formImport('form.en-us.lg')}
+${formImport('form-Choose.en-us.lg')}
+${propertyEntityValue()}
+${chooseEntity()}
+```
+
+# chooseEntity
+-```
+# ${entity}_chooseEntity()
+> Choose between '${entity}' values
+- \${chooseEnumEntity()}```
```

Comment:
>chooseEnumEntity [](http://example.com/codeflow?start=5&length=16)

I don't think this is right--this should be based on the dynamic list one, right?  Since the list is defined at runtime.